addons/website/static/src/scss/website.scss

```diff
@@ -181,6 +176,11 @@ $-seen-urls: ();
     }
 }
 
+#oe_main_menu_navbar:not(.o_hidden) + #wrapwrap .modal:not(.s_popup_bottom) {
+    // When the modal is in #wrapwrap, position top should be size of the navbar
+    top: $o-navbar-height;
+}
```

Comment:
I see this rule:
```scss
#oe_main_menu_navbar + #wrapwrap .o_header_affixed {
    top: $o-navbar-height;
}
```
They could be regrouped, with a placeholder selector to `@extend` or not